src/views/layout/template.njk

```diff
@@ -78,6 +78,264 @@
+                e = (n.src = e, cfg[u]);
+              return !e && "" !== e || "undefined" == n[u] || (n[u] = e),
+              n.onload = a,
+              n.onerror = i,
+              n.onreadystatechange = function (e, t) {
+                "loaded" !== n.readyState && "complete" !== n.readyState || a(0, t)
+              },
+              cfg.ld && cfg.ld < 0
+                ? x
+                  .getElementsByTagName("head")[0]
+                  .appendChild(n)
+                : setTimeout(function () {
+                  x
+                    .getElementsByTagName(I)[0]
+                    .parentNode
+                    .appendChild(n)
+                }, cfg.ld || 0),
+              n
+            };
+          T(k)
+        }
+        try {
+          f.cookie = x.cookie
+        } catch (p) {}
+        function t(e) {
+          for (; e.length;) 
+            !function (t) {
+              f[t] = function () {
+                var e = arguments;
+                d || f
+                  .queue
+                  .push(function () {
+                    f[t].apply(f, e)
+                  })
+              }
+            }
+          (e.pop())
+        }
+        var r,
+          s,
+          n = "track",
+          o = "TrackPage",
+          c = "TrackEvent",
+          n = (t([
+            n + "Event",
+            n + "PageView",
+            n + "Exception",
+            n + "Trace",
+            n + "DependencyData",
+            n + "Metric",
+            n + "PageViewPerformance",
+            "start" + o,
+            "stop" + o,
+            "start" + c,
+            "stop" + c,
+            "addTelemetryInitializer",
+            "setAuthenticatedUserContext",
+            "clearAuthenticatedUserContext",
+            "flush"
+          ]), f.SeverityLevel = {
+            Verbose: 0,
+            Information: 1,
+            Warning: 2,
+            Error: 3,
+            Critical: 4
+          }, (l.extensionConfig || {}).ApplicationInsightsAnalytics || {});
+        return !0 !== l[E] && !0 !== n[E] && (t(["_" + (
+          r = "onerror")]), s = C[r], C[r] = function (e, t, n, i, a) {
+          var o = s && s(e, t, n, i, a);
+          return !0 !== o && f["_" + r]({
+            message: e,
+            url: t,
+            lineNumber: n,
+            columnNumber: i,
+            error: a,
+            evt: C.event
+          }),
+          o
+        }, l.autoExceptionInstrumented = !0),
+        f
+      }(cfg.cfg),
+      (C[n] = i).queue && 0 === i.queue.length
+        ? (i.queue.push(e), i.trackPageView({}))
+        : e();
+    })({
+      src: "https://js.monitor.azure.com/scripts/b/ai.3.gbl.min.js",
+      // name: "appInsights",
+      // ld: 0,
+      // useXhr: 1,
+      crossOrigin: "anonymous",
+      // onInit: null,
+      // cr: 0,
+      cfg: { // Application Insights Configuration
+        connectionString: "{{ appInsightsConnectionString }}"
+      }
+    });
```

Comment:
could you please move this scripts to a js file and import it?
you can use this script tag just to assign appInsightsConnectionString to a variable, as it is global one it should be available inside the js file